graphql_compiler/tests/snapshot_tests/test_cost_estimation.py

```diff
@@ -0,0 +1,847 @@
+# Copyright 2019-present Kensho Technologies, LLC.
+from datetime import date
+import unittest
+
+import pytest
+
+from .. import test_input_data
+from ...compiler.metadata import FilterInfo
+from ...cost_estimation.cardinality_estimator import estimate_query_result_cardinality
+from ...cost_estimation.filter_selectivity_utils import (
+    ABSOLUTE_SELECTIVITY, FRACTIONAL_SELECTIVITY, Selectivity, _combine_filter_selectivities,
+    _get_filter_selectivity
+)
+from ..test_helpers import generate_schema_graph
+
+
+def create_lookup_counts(count_data):
+    """Create lookup_counts function for use in estimating query cost."""
+    def lookup_counts(name):
+        """Lookup the total number of instances and subinstances of a given class name."""
+        return count_data[name]
+    return lookup_counts
+
+
+# The following TestCase class uses the 'graph_client' fixture
```

Comment:
I think you might have already renamed this fixture in `master` so this merge might fail to apply correctly.